tests/wpt/metadata/websockets/Close-1000-reason.any.js.ini

```diff
@@ -0,0 +1,17 @@
+[Close-1000-reason.any.html]
```

Comment:
That's the weird thing where none of the WebSockets tests pass, I'll revert those. Is upstream even aware of this problem? Is it something that we ourselves broke somehow in our setup?

Reply:
I have not seen anybody else complaining about this.

Reply:
Did we document the failure somewhere? Looking for the output and maybe stacktraces and whatnot.

Reply:
https://github.com/servo/servo/issues/20734. https://github.com/w3c/web-platform-tests/pull/10804 was reverted before we got a chance to observe any output changes because it broke gecko's setup.